### DISCUSSION TITLE
Pre-X11 windowing system (MGR)

### DISCUSSION THREAD

**Discussion by Randrianasulu** [jbruchon/elks#1567] [General]
Probably not 16-bit friendly, but I thought it was cool

https://hack.org/mc/mgr/

https://m.youtube.com/watch?v=atvTV9xU3Cw&t=20s
at 3:06

**Comment by ghaerr** [comment]
Pretty cool, thanks for the links. Are you aware that ELKS supports an early version of [The Nano-X Window System](https://github.com/ghaerr/microwindows) and has several binaries available that run on IBM PC EGA graphics cards and PC-98?

**Comment by Randrianasulu** [comment]
yeah, I tried nano-x tetris, but because I run it on CGA emulated machine under kernel compiled without cga support - it just hanged :)
